spec/models/artist_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Artist, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+    describe 'Artist' do
+     let(:attributes) {{
+         name: "Lady Gaga",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.